controllers/scraper_test.go

```diff
@@ -33,15 +33,15 @@ var _ = Describe("ScraperController", func() {
 		TruncateTable("user")
 	})
 
-	Describe("POST /scraper/keyword", func() {
+	Describe("POST /dashboard/search_keyword", func() {
```

Comment:
Ho w about renaming `controllers/scraper.go` to `controllers/dashboard.go`?

Reply:
I've move those function from scraper controller to the dashboard one ✌️